scripts/honcho_oauth_setup.py

```
@@ -0,0 +1,133 @@
+        device_code: str = device_data["device_code"]
+        user_code: str = device_data["user_code"]
+        verification_uri: str = device_data["verification_uri"]
+        interval: int = device_data.get("interval", 5)
+        expires_in: int = device_data.get("expires_in", 300)
+
+        print()
+        print("=" * 60)
+        print("  Honcho OAuth Setup")
+        print("=" * 60)
+        print()
+        print(f"  1. Open this URL in your browser:")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)